src/ts/components/dashboard/Dashboard.tsx

```diff
@@ -20,9 +20,13 @@ import { KeyringPair$Json } from '@polkadot/keyring/types'
 import Balance from '../account/Balance'
 import Identicon from 'polkadot-identicon'
 import { saveSettings } from '../../background/store/settings'
+import { Popup } from 'semantic-ui-react'
```

Comment:
It is a duplicate import

Reply:
i will push a fix to this branch to fix it along with my css change

Reply:
👍